securedrop/manage.py

```diff
@@ -178,7 +179,7 @@ def _add_user(is_admin=False):
                                                   exc_traceback)))
         return 1
     else:
-        print('User "{}" successfully added'.format(username))
+        print('Admin "{}" successfully added'.format(username))
```

Comment:
[also this line](https://github.com/freedomofpress/securedrop/pull/2476#discussion_r146936772)